library/options.inc.php

```diff
@@ -87,7 +87,7 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
             $tag_name_esc = $tag_name_esc . "[]";
         }
 
-	$s .= "<select name='$tag_name_esc'";
+  $s .= "<select class='form-control' name='$tag_name_esc'";
```

Comment:
note there is a parameter that brings in class, so this would break if that parameter was used. Instead need to change the code at line 104 to something like:
``````
	if (!empty($class)) {
        $class_esc = attr($class);
        $s .= " class='form-control $class_esc'";
	}
    else {
        $s .= " class='form-control'";
    }